### ISSUE TITLE
[Feature Request] High-resolution OpenGL renderer 

### ISSUE THREAD

**Issue by lonelycare** [libretro/mgba#287]
With this enabled the GBA screen is rendered on the GPU instead of the usual software rendering, which allows scaling up beyond the usual 240×160 resolution without thrashing the CPU. 

This lets rotation and scaling effects look much cleaner and allows the “mode 7” effect to be even more impressive.

**Comment by hizzlekizzle** [comment]
Yeah, we've discussed this some and the general reduction in portability moving from software rendering to hardware rendering is a significant drawback.

**Comment by StardustPassingBy** [comment]
Seconding this - the difference for Super Circuit in particular is massive. The standalone version works for this but doesn't have RetroAchievements support, and it sucks having to choose between one or the other.

**Comment by MattiasN86** [comment]
Wouldn't it be possible to also add to be able to switch from software rendering and hardware rendering in settings?
